packages/farms/tests/v3.test.ts

```diff
@@ -1,26 +1,45 @@
-import { ChainId, chainNames } from '@pancakeswap/chains'
+import { ChainId, chainNames, isTestnetChainId } from '@pancakeswap/chains'
 import { Pool } from '@pancakeswap/v3-sdk'
 import groupBy from 'lodash/groupBy'
+import { isAddressEqual } from 'viem'
 import { describe, expect, it } from 'vitest'
 import { priceHelperTokens } from '../constants/common'
 import { farmsV3ConfigChainMap } from '../constants/v3'
+import { supportedChainIdV3 } from '../src'
 import { CommonPrice, getFarmsPrices } from '../src/fetchFarmsV3'
 
-describe('Config farms V3', () => {
-  Object.entries(farmsV3ConfigChainMap).forEach(([_chainId, farms]) => {
+const tokenListMap = {
+  [ChainId.BSC]: 'https://tokens.pancakeswap.finance/pancakeswap-extended.json',
+  [ChainId.ETHEREUM]: 'https://tokens.pancakeswap.finance/pancakeswap-eth-default.json',
+  [ChainId.ZKSYNC]: 'https://tokens.pancakeswap.finance/pancakeswap-zksync-default.json',
+  [ChainId.POLYGON_ZKEVM]: 'https://tokens.pancakeswap.finance/pancakeswap-polygon-zkevm-default.json',
+  [ChainId.ARBITRUM_ONE]: 'https://tokens.pancakeswap.finance/pancakeswap-arbitrum-default.json',
+  [ChainId.LINEA]: 'https://tokens.pancakeswap.finance/pancakeswap-linea-default.json',
+  [ChainId.BASE]: 'https://tokens.pancakeswap.finance/pancakeswap-base-default.json',
+  [ChainId.OPBNB]: 'https://tokens.pancakeswap.finance/pancakeswap-opbnb-default.json',
+}
+
+describe('Config farms V3', async () => {
+  const tokenListByChain = {}
+
+  for await (const [chainId, url] of Object.entries(tokenListMap)) {
```

Comment:
`Object.entries(tokenListMap)` doesn't return async iterable? We can remove the `await` here?